### PR TITLE
84 landing page prototype

### PR DESCRIPTION
**Pull request by absurd-turtle**
## Basics

- [x] Details of what you changed are in commit messages
      (first line should have `module: short statement` syntax)
- [ ] References to issues, e.g. `close #X`, are in the commit messages.
- [x] The buildserver is happy.
- [x] The PR is rebased with current master.

## Checklist

- [ ] I added unit tests for my code
- [ ] I fully described what my PR does in the documentation
      (not in the PR description)
- [x] I fixed all affected documentation
- [x] I fixed all affected decisions
- [x] I added code comments, logging, and assertions as appropriate
- [x] I mentioned [every code](/.reuse/dep5) not directly written by me in [reuse syntax](https://reuse.software/)

## Review

- [ ] Documentation is conforming to [our Documentation Guidelines](/doc/documentation.md)
- [ ] Examples are well chosen and understandable
- [ ] Code is conforming to our Coding Guidelines
- [ ] Code is consistent to [our Design Decisions](/doc/decisions)

### CODE REVIEW

**Comment by markus2330** [comment]
@yvonnemarkl really likes your logos!!! :rocket:

File: frontend/src/features/landing_page/routes/LandingPage.tsx [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+      {/* <img src="https://nextcloud.markus-raab.org/nextcloud/index.php/apps/files_sharing/publicpreview/BLfGxHDZxrF4NKo?file=/20230204_Entwurf_Yvonne.jpeg&fileId=466982&x=1080&y=1920&a=true" /> */}
+
+      <section className="m-8" id="gallery">
+        <h2 className="mt-12 ml-8 border-l-2 pl-4">Gallery</h2>
+        <PhotoGallery />
+      </section>
+      <footer className="flex justify-center gap-4 bg-white p-4 px-8">
+        <a
+          className="border-b border-stone-700"
+          href="https://github.com/ElektraInitiative/PermaPlant/"
```
Comment (markus2330) [inline frontend/src/features/landing_page/routes/LandingPage.tsx:71]:
```suggestion
          href="https://github.com/ElektraInitiative/Permaplant/"
```

File: frontend/src/routes/index.ts [hunk 2]
```diff
@@ -12,6 +13,11 @@ const routes: Routes = {
     path: '/seeds',
     title: 'Meine Saatgüter',
   },
+  [Pages.LandingPage]: {
+    component: LandingPage,
+    path: '/',
+    title: 'PermaPlanT',
```
Comment (markus2330) [inline frontend/src/routes/index.ts:19]:
```suggestion
    title: 'PermaplanT',
```

**Review by markus2330 — CHANGES_REQUESTED** [review]
PermaplanT is sometimes wrongly capitalized, more info comes in an issue.

**Comment by markus2330** [comment]
Please tell us when we can take a look. We are already very curios! :heart:

**Comment by absurd-turtle** [comment]
> Please tell us when we can take a look. We are already very curios! heart

You can have a look at it again. I have implemented a dark mode(almost complete), did some structural changes to the UI (colors, button component) and tried to include your suggestions.

**Comment by markus2330** [comment]
@buenaflor would be great if you can make a short review, then you maybe need to fix less later :wink:

**Review by markus2330 — CHANGES_REQUESTED** [review]
Please rebase and add documentation for the components you changed.

**Comment by absurd-turtle** [comment]
> Please rebase and add documentation for the components you changed.

done

**Comment by absurd-turtle** [comment]
> Please rebase and add documentation for the components you changed.

done

**Review by markus2330 — APPROVED** [review]

**Pull request merged**